src/framework/pagination/_pagination.scss

```diff
@@ -0,0 +1,46 @@
+.pagination {
+  line-height: 1;
+  margin: 1em auto;
+  text-align: center;
+}
+.pagination ul {
+  -moz-user-select: none;
+  -khtml-user-select: none;
+  -webkit-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+
+  list-style-type: none;
+  display: inline;
+  font-size: 100%;
+  margin: 0;
+  padding: .5em;
+}
+
+.pagination ul li.pagination__enabled {
+  cursor: pointer;
+}
+
+.pagination ul li {
+  display: inline-block;
+  border-radius: 3px;
+}
+
+.pagination ul li.pagination__page {
```

Comment:
I would rather make  these nested as it's sass, 

```scss
.pagination ul {
  li.pagiation_enabled{ 
    margin: 0;
  }

  i.pagination__page{ 
    margin: 1px;
  }
}
```

Reply:
Yeah  `_organizationSwitcherItem.scss` has crossed my eyes, I thought you were not using `scss` in this way